### ISSUE TITLE
Voyager media manager not creating folder if storage set to Azure

### ISSUE THREAD

**Issue by zeeshanch1118** [thedevdojo/voyager#5178]
## Version information

 - Laravel: v^7.24
 - Voyager: v^0.2.9
 - PHP: 7.4
 - Database: MySql

## Description

I have setup disk storage to Azure.

By default when I'm uploading the images files, it works fine i.e. creating automatically year/month folder and file uploads inside. But when I try to create a folder myself by going to option Add Folder, it doesn't create. Although it alerts as successfully created. But it doesn't show anywhere.

When I change disk storage to public (instead of azure) then folders are creating perfectly as expected. How this should be fixed for Azure?

**Comment by fletch3555** [comment]
Please check your Voyager version and correct.  0.2.9 is almost certainly not right.

Also, please provide some basic troubleshooting steps. Check the laravel logs, browser console, etc for errors and provide them if found.

Finally, please provide the storage disk configuration (with secrets obfuscated) you're using, and any relevant 3rd party packages (with version) you're using for this.

**Comment by zeeshanch1118** [comment]
From terminal I could see this: tcg/voyager  1.x-dev 5753a2c by running command `"composer show -i"`

It doesn't show anything in browser console, nor the laravel logs. 

In filesystems.php below are my configurations:

```
'default' => env('FILESYSTEM_DRIVER', 'azure'),

'azure' => [
            'driver' => 'azure',
            'name' => env('AZURE_STORAGE_NAME'),
            'key' => env('AZURE_STORAGE_KEY'),
            'container' => env('AZURE_STORAGE_CONTAINER'),
            'url' => env('AZURE_STORAGE_URL'),
            'prefix' => null,
        ],
```

Credentials for above Azure are already correctly set in .env file.

And as I said, everything works fine when a file is uploaded from Media Manager or any field of bread. But when I try to create folder myself directly from Media Manager, it doesn't. But when above configurations are changed to public i.e.:

`'default' => env('FILESYSTEM_DRIVER', 'public'),`

It also starts creating folder as well. But I want to make default storage on Azure and need to create folders.

Please help me to resolve this issue.

**Comment by fletch3555** [comment]
Azure is not a supported filesystem driver in laravel, so again, please identify what package you're using and what version of it.

Also, any reason you're using the `1.x-dev` version of Voyager instead of a released version? We don't guarantee stability in the dev versions, and therefore don't recommend it unless you're willing to dive into Voyager code to debug and assist with submitting PRs with fixes.

**Comment by zeeshanch1118** [comment]
Okay let me try to update Voyager and see if this fixes. 

I actually taken over already developed project for a client that's why didn't change anything.

**Comment by zeeshanch1118** [comment]
I tried to updated and in composer.json I can see below:

"pvtl/voyager-blog": "^0.2.9",
"tcg/voyager": "^1.4"

**Comment by zeeshanch1118** [comment]
Actually I'm able to upload files successfully on Azure and I can see uploaded files there too, for example this is https://owosdevapp3container.blob.core.windows.net/owosdevapp3store/posts/December2020/c8IEx9mSOnIjLwRCOTZW.jpg one of the files which is uploaded by Voyager's Media Manager. I just have issue for creating new folders in Media Manager, otherwise everything works fine.

**Comment by fletch3555** [comment]
We don't maintain pvtl/voyager-blog, but that does explain where you got the 0.2.9 version from

**Comment by zeeshanch1118** [comment]
Please watch this video: https://share.vidyard.com/watch/z1T8CRV3RR7gLrxzfKzZgA?

**Comment by fletch3555** [comment]
No, I understand the problem you're seeing. But I don't have enough information to even attempt to reproduce it.  I need you to do more debugging and prove the requested information.

**Comment by zeeshanch1118** [comment]
The strange thing is that when I change storage to Public, everything works fine including adding new folders. Just doesn't work when default storage is changed to Azure.
